src/consensus/vote.hpp

```diff
@@ -251,4 +191,82 @@ class NextVotesForPreviousRound {
   LOG_OBJECTS_DEFINE
 };
 
+struct votesBundle {
```

Comment:
types start with capital letter

Reply:
Fixed start capital letter, Thanks!
In my opinion, struct is better for creating value types, and class is better for making use of features. I prefer struct here